.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -9,24 +9,18 @@ body:
   - attributes:
       label: "Preflight checklist"
       options:
-        - label:
-            "I could not find a solution in the existing issues, docs, nor
+        - label: "I could not find a solution in the existing issues, docs, nor
```

Comment:
Should be use yaml multiline strings here?

Reply:
That's a good idea. This file is synced from https://github.com/ory/meta, I have created an [issue there](https://github.com/ory/meta/issues/189) to track this idea since it's a separate and larger change.

Reply:
Playing around with this, I don't think multi-line strings are a good idea here because we combine three tools here (YML, Markdown inside the strings, and Prettier). With YML multi-line strings, you have two options. They both have problems in our situation.

1. keep all newlines (block style): prevents reflowing text to match the max line length, we end up with really long lines or have to split lines manually --> bikeshed debates
2. replace newlines with spaces (block chomping): removes all newlines and thereby also the empty lines that separate paragraphs in Markdown --> format of text on github.com UI is garbled

See http://yaml-multiline.info for examples. 

This is a good example for how the Unix principle to split up tooling into many "sharp tools" leads to wonderful flexibility while prototyping but rarely the optimal result. :slightly_smiling_face:

Reply:
Since these files come from the `meta` repo, I am ignoring them here and fix the formatting in meta.